projects/js-packages/charts/src/hooks/test/use-chart-margin.test.tsx

```diff
@@ -102,7 +102,8 @@ describe( 'useChartMargin', () => {
 		const height = 300;
 		const theme = baseTheme;
 		const { result } = renderHook( () => useChartMargin( height, options, data, theme ) );
-		expect( result.current.top ).toBe( 20 );
+		// 12px font → ~15px line-height + 8 tick length + 8 padding = 31
+		expect( result.current.top ).toBe( 31 );
 		expect( result.current.bottom ).toBe( 10 );
```

Comment:
FAIL  src/hooks/test/use-chart-margin.test.tsx
  ● useChartMargin › sets top and bottom margin for top x axis

    expect(received).toBe(expected) // Object.is equality

    Expected: 10
    Received: 20

      105 |             // 12px font → ~15px line-height + 8 tick length + 8 padding = 31
      106 |             expect( result.current.top ).toBe( 31 );
    > 107 |             expect( result.current.bottom ).toBe( 10 );
          |                                             ^
      108 |     } );
      109 |
      110 |     it( 'returns default margin if no axis options', () => {

      at Object.toBe (src/hooks/test/use-chart-margin.test.tsx:107:35)

Reply:
Feel like bottom should be 20 here, but I'll leave it to @chihsuan for a second opinion.